src/app/auth/login.tsx

```diff
@@ -1,15 +1,21 @@
 import { Link, router } from 'expo-router';
 import React, { useState } from 'react';
-import { Alert, View } from 'react-native';
-import { Button, Input } from 'react-native-elements';
+import { Text, View, StyleSheet, SafeAreaView } from 'react-native';
```

Comment:
Update your SafeAreaView import to be from react-native-safe-area-context instead of react-native, because the current import doesn't work on android.